build.sbt

```diff
@@ -122,7 +126,8 @@ lazy val cliPackSettings = Def.settings(
       "auxlib",
       "scalalib"
     ).map { lib =>
-      val nativeBinVersion = ScalaNativeCrossVersion.binaryVersion(snVer)
+      val nativeBinVersion =
+        ScalaNativeCrossVersion.binaryVersion(snVer.stripSuffix("-SNAPSHOT"))
```

Comment:
This part is interesting I think, this might be a bug in Scala Native sbt plugin, I had to remove "-SNAPSHOT" manually, otherwise the binary version would be parsed as 0.4.3-SNAPSHOT, leading to errors when resolving dependencies

Reply:
Yes, I've seen this one in the past, it would probably need some fix in core Scala Native, however it seems that for published snapshots the binary version was correct, which is strange